css.md

```diff
@@ -0,0 +1,177 @@
+1. [Property Order](#property-order)
+1. [Naming Conventions](#naming-conventions)
+1. [Property values](#property-values)
+1. [Media Queries](#media-queries)
+1. [Less](#less)
+    - [Less General Practices](#less-general-practices)
+    - [Selectors and Nesting](#selectors-and-nesting)
+    - [Variables](#variables)
+
+## General Practices
+
+- Use soft-tabs with a two space indent
+- One line per selector
+- Always end property declarations with a semicolon
+- Put spaces after `:` in property declarations
+- Put spaces before `{` in rule declarations
+- Place `{` on the same line as the (last) selector and `}` on its own line
+- 
+
+## Property Order
+
+Box model properties go first, in order from inside the box out, followed by positioning.
+
+Remaining properties are in alphabetical order.
+
+```css
+.selector {
+    /* Display & Box Model */
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100px;
+    height: 100px;
+    padding: 10px;
+    border: 10px solid #333;
+    margin: 10px;
+    overflow: hidden;
+
+    /* Positioning */
+    float: left;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 10;
+
+    /* Other */
+    background: #000 url('../img/bg.png') no-repeat center top;
+    color: #fff;
+    cursor: pointer;
+    font-family: sans-serif;
+    font-size: 16px;
+    font-style: italic;
+    font-weight: bold;
+    line-height: 1;
+    list-style: none;
+    text-align: right;
+}
+```
+
+_N.B.: It is not necessary to include the group headings in your actual CSS. They are just for clarity in this example._
+
+## Naming Conventions
+Custom [BEM ("Block Element Modifier")](https://en.bem.info/method/definitions/) format:
+
+```css
+.block-name
+.block-name_element-name
+.block-name__block-modifier
+.block-name_element-name__element-modifier
+```
+
+### Avoid creating elements of modifiers
+Appending an element name to a modifier class can result in a confusing class name like `.list__space_item`. 
+Avoid this in favor of using a descendant, like this: `.list__spaced .list_item`.
+
+### `id` attribute
+
+While the `id` attribute might be fine in HTML and JavaScript, it should be **avoided entirely** inside stylesheets for a few reasons:
+
+- ID selectors are not reusable
+- Can lead to priority issues
+
+##### Good
+
+```css
+.ur-name {
+  // ...
+}
+```
+
+##### Bad
+
+```css
+#ur-name {
+  // ...
+}
+```
+
+Just assign a class name to the element.
+
+## Property values
+
+These rules apply to your CSS property values
+
+- If the value of a property is `0`, do not specify units
+- The `!important` rule should be aggressively avoided.
+  - Keep style rules in a sensible order
+  - Compose styles to dissipate the need for an `!important` rule
+  - Fine to use in limited cases
+    - Overlays
+    - Declarations of the `display: none !important;` type
+- Use hex color codes `#000` unless there's an explicit need for an `rgba` declaration
```

Comment:
Most likely true, but as mentioned elsewhere, this document should be valuable for plain Jane CSS, too (excepting the section on Less). The bullet for color variables in the Less section could probably use a little elaboration.